packages/stylelint-config/rules/stylelint-config-lmc.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  // Package `@lmc-eu/stylelint-config/optional` contains ruleset for CSS properties order.
+  extends: ['@lmc-eu/stylelint-config/optional'].map(require.resolve),
```

Comment:
I'm unsure about reason why the package that does only check for CSS properties order is called `optional`, but I figure this is a historical reason since earlier this ruleset was generally perceived as high level restriction and abusing. I hope I can say that this is not reality nowadays anymore so I'm trying to make it LMC standard.

Please give feedback whether you think it is acceptable BC or not.

Reply:
The intention of an `optional` ruleset in all packages in this repo is generally considered a high-level restriction, as you mention. It should be more strict or there are rules that are not generally accepted by most developers or apps but are considered a good practice and should be accepted maybe in near future.

In this case, there was a debate with @adamkudrna that this ordering goes against community standards and should be changed. If I remember him well.

Maybe for this time should be better to just rename it to `order.js`? Because it only maintains the order rules.

Reply:
Conversation solved by internal discussion.